e2e/onboarding-create.e2e-spec.ts

```diff
@@ -1,9 +1,12 @@
+import { browser } from 'protractor';
+
 import { OnboardingCreatePage } from './onboarding-create.po';
 
 describe('Onboarding Create', () => {
   let page: OnboardingCreatePage;
 
   beforeEach(() => {
+    browser.waitForAngularEnabled(false);
```

Comment:
Without adding browser.waitForAngularEnabled(false) all our e2e tests are failed because setInterval() function is used to calculate last updated time.
Due to it is used in the header component it is affected all pages.

Why is this changes being made in several test files? - 'waitForAngularEnabled' is a configuration of browser and it's more than enough to add it in the first test file (onboarding-create.e2e-spec.ts), just because it is run firstly. We decided to added it to each test file just to make them independent from each other. We will remove 'waitForAngularEnabled' from all files except onboarding-create.e2e-spec.ts